download-maven-plugin/.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: java
+sudo: false
+cache:
+  directories:
+    - $HOME/.m2
+env:
+  global:
+    - MAVEN_OPTS="-XX:MaxPermSize=256m -Xmx512m"
+    - JAVA_OPTS="-XX:MaxPermSize=256m -Xmx512m"
+jdk:
+  - oraclejdk7
```

Comment:
@mickaelistria I know many issues based on jdk type. Prebuild for both will show us those issues rapidly.